File: Java/src/test/java/org/sammancoaching/HtmlPagesConverterTest.java
package org.sammancoaching;

import org.junit.jupiter.api.Test;

import java.io.IOException;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class HtmlPagesConverterTest {

    @Test
    public void foo() throws IOException {
        HtmlPagesConverter converter = new HtmlPagesConverter("foo");
        assertEquals("fixme", converter.getFilename());
    }

}

File: kotlin/src/test/kotlin/com/katas/HtmlPagesConverterTest.kt
package com.katas

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class HtmlPagesConverterTest {
    @Test
    fun foo() {
        val converter = HtmlPagesConverter("foo")
        assertThat(converter.filename).isEqualTo("fixme")
    }
}
